diff --git a/.overlay/etc/pacman.conf b/.overlay/etc/pacman.conf
index 0a04238..d74f97d 100644
--- a/.overlay/etc/pacman.conf
+++ b/.overlay/etc/pacman.conf
@@ -89,8 +89,8 @@ Include = /etc/pacman.d/mirrorlist
 #[multilib-testing]
 #Include = /etc/pacman.d/mirrorlist
 
-#[multilib]
-#Include = /etc/pacman.d/mirrorlist
+[multilib]
+Include = /etc/pacman.d/mirrorlist
 
 # An example of a custom package repository.  See the pacman manpage for
 # tips on creating your own repositories.
